src/sage/graphs/generic_graph.py

```diff
@@ -1709,9 +1709,20 @@ def igraph_graph(self, vertex_list=None, vertex_attrs={}, edge_attrs={}):
             True
             sage: G.edges(sort=True) == H.edges(sort=True)             # optional - python_igraph
             False
+
+        Check input parameter ``vertex_list``::
+
+            sage: G = Graph(3)
+            sage: G.igraph_graph(vertex_list=[])                       # optional - python_igraph
+            Traceback (most recent call last):
+            ...
+            ValueError: ``vertex_list`` must be a permutation of the vertices
```

Comment:
please no rst syntax in error messages

Reply:
I have removed rst syntax from many errors messages.